app.js

```diff
@@ -1,53 +1,52 @@
-function getDate() {
-	var date = document.getElementById("date"); 
-	date.innerHTML = (new Date()).toString().split(' ').splice(0,4).join(' ');
+function loadDate() {
+  var currentDate = new Date();
+  var dateString = currentDate.toString()
+                     .split(" ")
+                     .splice(0, 4)
+                     .join(" ");
```

Comment:
If you decide to go for this formatted style for `dateStyle`, make sure to explain how function chaining works.